test/ivToken/IvToken.t.sol

```diff
@@ -0,0 +1,157 @@
+        // when
+        // then
+        assertEq(decimals, 18);
+    }
+
+    function testShouldTransferOwnership() public {
+        // given
+        IvToken ivTokenDai = new IvToken("IV DAI", "ivDAI", address(_daiMockedToken));
+        address ownerBefore = ivTokenDai.owner();
+        // when
+        ivTokenDai.transferOwnership(_userOne);
+        vm.prank(_userOne);
+        ivTokenDai.confirmTransferOwnership();
+        // then
+        address ownerAfter = ivTokenDai.owner();
+        assertEq(ownerBefore, _admin);
+        assertEq(ownerAfter, _userOne);
+    }
+
+    function testShouldNotTransferOwnershipWhenSenderNotCurrentOwner() public {
+        // given
+        IvToken ivTokenDai = new IvToken("IV DAI", "ivDAI", address(_daiMockedToken));
+        address ownerBefore = ivTokenDai.owner();
+        // when
+        vm.prank(_userOne);
+        vm.expectRevert(abi.encodePacked("Ownable: caller is not the owner"));
+        ivTokenDai.transferOwnership(_userOne);
+        // then
+        address ownerAfter = ivTokenDai.owner();
+        assertEq(ownerBefore, _admin);
+        assertEq(ownerAfter, _admin);
+    }
+
+    function testShouldNotConfirmTransferOwnershipWhenSenderNotAppointedOwner() public {
+        // given
+        IvToken ivTokenDai = new IvToken("IV DAI", "ivDAI", address(_daiMockedToken));
+        address ownerBefore = ivTokenDai.owner();
+        // when 
+        ivTokenDai.transferOwnership(_userOne);
+        vm.prank(_userTwo);
+        vm.expectRevert(abi.encodePacked("IPOR_007"));
+        ivTokenDai.confirmTransferOwnership();
+        // then
+        address ownerAfter = ivTokenDai.owner();
+        assertEq(ownerBefore, _admin);
+        assertEq(ownerAfter, _admin);
+    }
+
+    function testShouldNotConfirmTransferOwnershipTwiceWhenSenderNotAppointedOwner() public {
+        // given
+        IvToken ivTokenDai = new IvToken("IV DAI", "ivDAI", address(_daiMockedToken));
+        address ownerBefore = ivTokenDai.owner();
+        ivTokenDai.transferOwnership(_userOne);
+        vm.prank(_userOne);
+        ivTokenDai.confirmTransferOwnership();
+        // when
+        vm.expectRevert(abi.encodePacked("Ownable: caller is not the owner"));
+        ivTokenDai.transferOwnership(_userOne);
+        // then
+        address ownerAfter = ivTokenDai.owner();
+        assertEq(ownerBefore, _admin);
+        assertEq(ownerAfter, _userOne);
+    }
+
+    function testShouldNotTransferOwnershipWhenSenderAlreadyLostOwnership() public {
+        // given
+        IvToken ivTokenDai = new IvToken("IV DAI", "ivDAI", address(_daiMockedToken));
+        address ownerBefore = ivTokenDai.owner();
+        // when 
+        ivTokenDai.transferOwnership(_userOne);
+        vm.prank(_userOne);
+        ivTokenDai.confirmTransferOwnership();
+        // then
+        vm.prank(_admin);
+        vm.expectRevert(abi.encodePacked("Ownable: caller is not the owner"));
+        ivTokenDai.transferOwnership(_userOne);
```

Comment:
This line should be in when a section 
lines 111-117  -> // given
lines 119-121 -> // when